code/game/objects/items/storage/uplink_kits.dm

```diff
@@ -195,6 +195,15 @@
 			new /obj/item/dnainjector/cryokinesis(src)
 			new /obj/item/gun/energy/temperature/security(src)
 			new /obj/item/melee/transforming/energy/sword/saber/blue(src) //see see it fits the theme bc its blue and ice is blue
+			
+		if("ancient") //A kit so old, it's probably older than you.
+			new /obj/item/card/emag(src)
+			new /obj/item/pen/sleepy(src)
+			new /obj/item/reagent_containers/pill/cyanide
+			new /obj/item/chameleon(src)
```

Comment:
you might want to mention in a comment here that this is a substitute for the cloaker